core/src/main/java/org/infinispan/protostream/impl/TagWriterImpl.java

```diff
@@ -131,13 +131,30 @@ public void writeVarint64(long value) throws IOException {
       encoder.writeVarint64(value);
    }
 
+   public static boolean onlyUTF8(String ba, int off, int len) {
```

Comment:
Shouldn't this method be called `isAsciiOnly ()` ?

Reply:
Yeah, I didn't push my update ;)